pkg/ebpf/debugfs.go

```diff
@@ -16,16 +16,26 @@ import (
 	"regexp"
 	"strconv"
 	"strings"
+	"sync"
+	"time"
 
+	"github.com/DataDog/datadog-agent/pkg/telemetry"
 	utilkernel "github.com/DataDog/datadog-agent/pkg/util/kernel"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 	manager "github.com/DataDog/ebpf-manager"
 )
 
-var myPid int
+const kProbeTelemetryName = "network_tracer_kprobes"
```

Comment:
There can be more than just network tracer kprobes installed on the system. I realize they were collected and reported under the `network_tracer` namespace before. Option 1 for this PR is to take all these stats, but filter them to just the NPM kprobes.

Option 2 would be to leave changing the reporting of these stats to my team, and we can figure out a way to report them for all of system-probe.

I personally would prefer option 2, because these stats really should be using the `prometheus.Collector` interface to get the values on demand, rather than refreshing on an interval that is likely out of sync with the telemetry collector.

Reply:
@leeavital @hmahmood WDYT?

Reply:
A rename for sure.